src/langcheck/metrics/zh/reference_based_text_quality.py

```diff
@@ -39,57 +36,36 @@ def semantic_similarity(
     with missing punctuation/different punctuation (one is declarative sentence,
     one is question) would lower the value to 0.2 ~ 0.3.
 
-    2. The 'openai' type, where we use OpenAI's 'text-embedding-ada-002' model
-    by default (this is configurable). See
-    `this example <https://langcheck.readthedocs.io/en/latest/metrics.html
-    #computing-metrics-with-openai-models>`__
-    for examples on setting up the OpenAI API key.
-
-    3. The 'azure_openai' type. Essentially the same as the 'openai' type,
-    except that it uses the AzureOpenAI client. Note that you must specify your
-    model deployment to use in ``openai_args``, e.g.
-    ``openai_args={'model': 'YOUR_DEPLOYMENT_NAME'}``
+    2. The EvalClient type, where you can use a similarlity scorer returned by
```

Comment:
Need to update `We currently support three embedding model types:` to `two` above